File: lingting-huawei/src/main/java/live/lingting/framework/huawei/HuaweiRequest.java
package live.lingting.framework.huawei;

import live.lingting.framework.http.HttpMethod;
import live.lingting.framework.http.api.ApiRequest;
import lombok.Getter;

/**
 * @author lingting 2024-09-12 21:43
 */
@Getter
public abstract class HuaweiRequest extends ApiRequest {

	@Override
	public HttpMethod method() {
		return HttpMethod.POST;
	}

	@Override
	public void onCall() {
		headers.contentType("application/json;charset=utf8");
	}

}

File: lingting-huawei/src/main/kotlin/live/lingting/framework/huawei/HuaweiRequest.kt
package live.lingting.framework.huawei

import live.lingting.framework.http.HttpMethod
import live.lingting.framework.http.api.ApiRequest

/**
 * @author lingting 2024-09-12 21:43
 */
abstract class HuaweiRequest : ApiRequest() {

    override fun method(): HttpMethod {
        return HttpMethod.POST
    }

    override fun onCall() {
        headers.contentType("application/json;charset=utf8")
    }

}
